### PR TITLE
[MIG] Port account_fiscal_position_vat_check to v8 and to new API

### PR DESCRIPTION
**Pull request by alexis-via**

### CODE REVIEW

File: account_fiscal_position_vat_check/__openerp__.py [hunk 1]
```diff
@@ -62,11 +58,6 @@
...
         'partner_view.xml',
     ],
-    'images': [
-        'images/fiscal_position_form.jpg',
-        'images/vat_check_invoice_validation.jpg',
-        ],
-    'installable': False,
-    'active': False,
+    'installable': True,
     'application': True,
```
Comment (pedrobaeza) [inline account_fiscal_position_vat_check/__openerp__.py]: Remove application flag

File: account_fiscal_position_vat_check/account_invoice.py [hunk 2]
```diff
@@ -20,39 +20,37 @@
...
-        'customer_must_have_vat': fields.boolean(
-            'Customer Must Have VAT number',
-            help="If enabled, OpenERP will check "
-                 "that the customer has a VAT number "
-                 "when the user validates a customer invoice/refund."
-        ),
-    }
+    customer_must_have_vat = fields.Boolean(
+        string='Customer Must Have VAT number',
+        help="If enabled, OpenERP will check that the customer has a VAT "
```
Comment (pedrobaeza) [inline account_fiscal_position_vat_check/account_invoice.py]: s/OpenERP/Odoo

File: account_fiscal_position_vat_check/partner.py [hunk 3]
```diff
@@ -42,8 +42,7 @@ def fiscal_position_change(self, cr, uid, ids, account_position,
                             "You have set the fiscal position '%s' "
                             "that require the customer to have a VAT number. "
                             "You should add the VAT number of this customer"
-                            " in OpenERP."
-                        ) % fp['name']
+                            " in OpenERP.") % fp.name
```
Comment (pedrobaeza) [inline account_fiscal_position_vat_check/partner.py]: s/OpenERP/Odoo

**Comment by pedrobaeza** [comment]
Some little remarks. The rest LGTM.

**Comment by pedrobaeza** [comment]
:+1: 

Thanks

**Comment by sebastienbeau** [comment]
Good work Alexis :+1:

**Comment by rafaelbn** [comment]
Sorry by asking here... could anyone say me the diference with odoo field vat_requirement please?



Thanks in advance for answering

**Comment by alexis-via** [comment]
There is no difference, we should use that new native field instead. The only "bad" reason is that the field in the OCA module existed before the native field was introduced. We should change this when we migrate the module to v9.

**Comment by rafaelbn** [comment]
OK, thanks @alexis-via

**Pull request merged**